WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -437,31 +435,132 @@ province_event = {
 			factor = 2.5
 			year = 750
 		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ca_wall_1 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ca_wall_2 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ca_wall_3 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ca_wall_4 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ca_wall_5 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ct_wall_3 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ct_wall_4 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = ct_wall_5 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = tp_wall_3 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = tp_wall_4 }
+		}
+		modifier = {
+			factor = 1.1
+			capital = { has_building = tp_wall_5 }
+		}
```

Comment:
Wouldn't it be simple to make this into an OR block?

Reply:
No, that would have a different effect. With my solution they stack.

Reply:
You can't have both level four and five in the same holding.

Reply:
I made a test event to check that before committing and it seems that lower levels still exist, but I can test that again of course.

Reply:
It is not how the code reads since it checks for a binary "is X level there or not", but if that is how it works fine.